internal/input/ledger_range.go

```diff
@@ -0,0 +1,159 @@
+package input
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/go/exp/ingest/ledgerbackend"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+type graphPoint struct {
+	Seq       int64
+	CloseTime time.Time
+}
+
+type graph struct {
```

Comment:
As with `graphPoint`, it'll be helpful to add a comment explaining this struct as well.